core/host.py

```diff
@@ -112,3 +115,18 @@ def safe_mk_dirs(path, print_res=False):
     if print_res:
         print(msg)
     os.makedirs(path, exist_ok=True)
+
+
+def validate_abi_files():
+    results = [
+        validate_abi(abi_filepath)
+        for abi_filepath in [
+            MANAGER_CONTRACTS_FILEPATH,
+            IMA_CONTRACTS_FILEPATH
+        ]
+    ]
+    if any(r['status'] == 'error' for r in results):
+        print('Some files are not exist or incorrect')
+        print_abi_validation_errors(results)
+    else:
+        print('Ok')
```

Comment:
Maybe we should add a more informative message?
Something like `ABI files validated successfully!`